tools/ci_build/github/linux/docker/scripts/training/requirements.txt

```diff
@@ -2,10 +2,10 @@
 -f https://download.pytorch.org/whl/torch_stable.html
 # transformers requires sklearn
 sklearn
-numpy==1.16.6
+numpy
```

Comment:
Why this line needs be changed?

Reply:
There are conflicts some conflicts between different numpy and tensorboard in different requirements.txt files. As the packaging pipeline runs 1) build, 2) test, 3) packaging and 4) documenting, all 4 requirements are needed to work together. numpy 1.16.6 is not compatible with all dependencies

Reply:
Last night I found training python packaging pipelines(including ROCM) uses a different numpy version than expected. If your code was built with numpy version A, then at runtime it requires at least number version >=A. From the requirements.txt in onnx runtime's top source dir, you can see we claim it requires numpy 1.16.6. I tried to increase it, but the change got rejected. 

I have reported this problem to Liqun and Suffian. They said it is ok for now but eventually it needs be fixed.

Reply:
You may resolve this conversation after reading this.

Reply:
I think we should not change this to release 1.8. It adds unnecessary risk. Maybe that was the context for the rejection?

I wouldnt recommend cherry-picking this PR to 1.8, but for the next release only

Reply:
Do we not need to pin the version number?

Reply:
I will try to pin it to 1.19.5